CMakeLists.txt

```diff
@@ -218,7 +218,7 @@ if(APPLE)
     HPCC_ADD_SUBDIRECTORY(package)
 endif(APPLE)
 
-if (NOT CLIENTTOOLS_ONLY)
+if (PLATFORM)
```

Comment:
This removes all the vcpkg include files from getting added to each plugin package.